corelib-db/pom.xml

```diff
@@ -13,8 +14,9 @@
     <artifactId>corelib-db</artifactId>
     <name>Europeana Core Library Database</name>
 
-    <organization><name>Europeana Foundation</name></organization>
     <url>http://www.europeana.eu</url>
+    <organization><name>Europeana Foundation</name></organization>
```

Comment:
not that I mind either way, but does that ordering of <organization> and <url> matter?

Reply:
There is a SonarQube rule that checks the ordering. It is at minor level though.